.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+jdk: oraclejdk8
+# Trusty dist needed for postgresql version 9.6
+dist: trusty
+# Caches downloaded libraries between builds
+cache:
+  directories:
+    - node_modules
+    - $HOME/.m2
+services:
+  - postgresql
+addons:
+  postgresql: '9.6'
+# Set the max_map_count for elasticsearch, as instructed by maven Elasticsearch plugin docs
+sudo: true
+before_script:
+  - sudo sysctl -w vm.max_map_count=262144
```

Comment:
I don't really understand what the setting does tbh, only saw in the travis docs that you cannot expect elasticsearch to start without it.
I think that means that when you follow the install SOP on the golden VM, the value also gets set to a high enough value.
I can think of two possible reasons why:
* It's a different OS, so perhaps the setting is already set by default on CentOS 
* Perhaps it gets set in the yum package.